### PR TITLE
Updated test on stories.py + small fixes to testing

### PR DESCRIPTION
**Pull request by alessandrobelfiore**

### CODE REVIEW

File: monolith/classes/test/test_die.py [hunk 1]
```diff
@@ -26,7 +26,3 @@ def test_multi_face_die(self):
     def test_die_to_str(self):
         die = Die(["1", "2", "3"], "test_theme")
         self.assertEqual(str(die), "['1', '2', '3']")
-
- 
-if __name__ == '__main__':
-    unittest.main()
```
Comment (sduoase) [inline monolith/classes/test/test_die.py]: don't we need these?

File: monolith/classes/test/test_dice_set.py [hunk 2]
```diff
@@ -35,6 +35,3 @@ def test_multi_die_multi_face(self):
         self.assertEqual(_throw_to_faces(dice_set.throw()), ["2", "5"])
         self.assertEqual(_throw_to_faces(dice_set.throw()), ["1", "5"])
         self.assertEqual(_throw_to_faces(dice_set.throw()), ["3", "5"])
- 
-if __name__ == '__main__':
-    unittest.main()
```
Comment (sduoase) [inline monolith/classes/test/test_dice_set.py]: same here

File: monolith/views/test/test_is_story_valid.py [hunk 3]
```diff
@@ -44,7 +44,3 @@ def test_case_sensitive(self):
         dice_roll = ["FOX", "dog", "bROwn"]
         story_text = "The quick BrOwN fox jumps over the lazy DOG"
         self.assertEqual(is_story_valid(story_text, dice_roll), True)
-    
- 
-if __name__ == '__main__':
-    unittest.main()
```
Comment (sduoase) [inline monolith/views/test/test_is_story_valid.py]: same here

File: monolith/views/test/test_stories.py [hunk 4]
```diff
@@ -66,15 +66,22 @@ def test_delete_story(self):
...
+        self._signup("fantastic@example.com", "betterNerfIrelia", "404", "404", "01/01/1964", True)
+        self._login("fantastic@example.com", "betterNerfIrelia")
+        reply = self.client.post('/stories/new_story', data={ "theme" : "Mountain", "dice_number" : "3"})
+        #TODO publish story first!
+        self._logout()
         #self._login("example@example.com", "admin")
         #reply = self.client.get('/random_story')
         #self.assert_template_used("story.html")
-        #self.assert_context("rolls_outcome", "[]")
+        #self.assert_context("message", "")
```
Comment (sduoase) [inline monolith/views/test/test_stories.py]: just remove them if we're not using/doing that

File: monolith/views/test/test_stories.py [hunk 5]
```diff
@@ -91,7 +98,83 @@ def test_new_story(self):
...
+        
+        # TODO
+        # success: published story
+        #reply = self.client.get('/write_story/2')
+        #self.assertEqual(reply.status_code, 200)
+        #reply = self.client.post('/write_story/2', data={
+        #         'text' : "1",
+        #         'title' : "ThisIsATitle",
+        #         'store_story' : 0})
+        #self.assert_template_used("/write_story.html")
```
Comment (sduoase) [inline monolith/views/test/test_stories.py]: same here

File: monolith/classes/test/test_die.py (hunk 1 above)
Comment (alessandrobelfiore) [reply monolith/classes/test/test_die.py]: The tests correctly run either way, but the coverage says these lines aren't tested, so I'm guessing they are not really needed

File: monolith/views/test/test_stories.py (hunk 5 above)
Comment (alessandrobelfiore) [reply monolith/views/test/test_stories.py]: Ok I was hoping to get it to work but I'll delete them for now

File: monolith/views/test/test_is_story_valid.py (hunk 3 above)
Comment (Gr3gbug) [reply monolith/views/test/test_is_story_valid.py]: me too notice the same thing

File: monolith/views/test/test_stories.py (hunk 4 above)
Comment (alessandrobelfiore) [reply monolith/views/test/test_stories.py]: Done

**Pull request merged**